src/settings-ui/Settings.UI/Views/VideoConference.xaml

```diff
@@ -49,10 +49,9 @@
                 <InfoBar
                     x:Uid="VideoConference_RunAsAdminRequired"
                     IsClosable="False"
-                    IsOpen="True"
+                    IsOpen="{Binding Mode=OneWay, Path=IsElevated}"
                     IsTabStop="True"
-                    Severity="Warning"
-                    Visibility="{Binding Mode=OneWay, Path=IsElevated, Converter={StaticResource ReverseBoolToVisibilityConverter}}" />
+                    Severity="Informational"/>
```

Comment:
Please keep "warning" as we use that on every admin info bar.

Reply:
Sorry, but I think this is more appropriate. Also, the infobar on File Explorer - "computer has to be restarted.." - is also Informational.

Reply:
General page > Admin setting.

Reply:
Maybe it's me, but I don't see it? 🙈